src/main/kotlin/com/openlattice/chronicle/storage/PostgresColumns.kt

```diff
@@ -19,5 +19,8 @@ class PostgresColumns {
         val FIRST_NAME = PostgresColumnDefinition("first_name", PostgresDatatype.TEXT)
         val LAST_NAME = PostgresColumnDefinition("last_name", PostgresDatatype.TEXT)
         val DATE_OF_BIRTH = PostgresColumnDefinition( "dob", PostgresDatatype.DATE )
+        val TUD_ID = PostgresColumnDefinition("tud_id", PostgresDatatype.UUID).notNull()
+        val SUBMISSION_DATE = PostgresColumnDefinition("submission_date", PostgresDatatype.TIMESTAMPTZ)
+        val TUD_DATA = PostgresColumnDefinition("tud_data", PostgresDatatype.JSONB)
```

Comment:
If we have submission date, consider calling this column just "submission" unless there is other non-submission data. @UnsungHero97 What do you think?

Reply:
This change makes sense to me especially since the column will only ever contain submission data.